apps/portal/examples/postgraphile-template/content/source/Dockerfile

```
@@ -0,0 +1,13 @@
+# Use the official PostGraphile image
+FROM graphile/postgraphile:latest
```

Hadolint (2.14.0)

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)